doc/developers/developing_scikit.rst

```diff
@@ -0,0 +1,671 @@
+.. _developing_scikit:
+
+==================================
+Developing scikit-learn estimators
+==================================
+
+Whether you are proposing an estimator for inclusion in scikit-learn,
+developing a separate scikit-learn package, or implementing custom
```

Comment:
"developing a separate scikit-learn package" is a bit confusing and I can't believe I suggested it! Replace with "developing a separate package compatible with scikit-learn"